Composer/packages/lib/shared/src/copyUtils/copyAdaptiveActionList.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { copyAdaptiveAction } from './copyAdaptiveAction';
+import { ExternalApi } from './ExternalApi';
+
+export async function copyAdaptiveActionList(actions, externalApi: ExternalApi): Promise<any[]> {
```

Comment:
```suggestion
export async function copyAdaptiveActionList(actions: MicrosoftIDialog[], externalApi: ExternalApi): Promise<MicrosoftIDialog[]> {
```

Reply:
This is resolved. Typed in commit 3bbc01c4.